backend/src/main/scala/com/softwaremill/adopttapir/util/Util.scala

```diff
@@ -0,0 +1,8 @@
+package com.softwaremill.adopttapir.util
```

Comment:
since this is not a class, I think the file name should be lower-cased

Reply:
Renamed.